pkg/security/probe/probe.go

```diff
@@ -1372,7 +1372,7 @@ func NewProbe(config *config.Config, statsdClient statsd.ClientInterface) (*Prob
 		erpcRequest:          &ERPCRequest{},
 		tcPrograms:           make(map[NetDeviceKey]*manager.Probe),
 		statsdClient:         statsdClient,
-		discarderRateLimiter: rate.NewLimiter(rate.Every(time.Second), 100),
+		discarderRateLimiter: rate.NewLimiter(rate.Every(time.Second/20), 100),
```

Comment:
can you add a comment about where these numbers come from? they just look at magic numbers to me cuz i have no context